src/router.js

```diff
@@ -1,19 +1,11 @@
-const fs = require('fs');
-const path = require('path');
+const homeHandler = require('./handlers/homeHandler');
 
 const router = (req, res) => {
   const endpoint = req.url;
   if (endpoint === '/') {
-    const filePath = path.join(__dirname, '..', 'public', 'index.html');
-    fs.readFile(filePath, (err, file) => {
-      if (err) {
-        res.writeHead(500, { 'Content-Type': 'text/html' });
-        res.end('<h1> Error 500: Internal Server Error! </h1>');
-      } else {
-        res.writeHead(200, { 'Content-Type': 'text/html' });
-        res.end(file);
-      }
-    });
+    homeHandler(req, res);
+  } else if (endpoint === '/index') {
+    homeHandler(req, res);
```

Comment:
you can use or operator instead of else if since in both cases you will do the same thing